File: official/docs/ruby/address/strict_verify_address_on_create.rb
require 'easypost'

EasyPost.api_key = ENV['EASYPOST_API_KEY']

strict_verifiable_address = EasyPost::Address.create(
  verify_strict: ['delivery'],
  street1: '417 montgomery streat',
  street2: 'FL 5',
  city: 'SAN FRANCISCO',
  state: 'CA',
  zip: '94104',
  country: 'US',
  company: 'EasyPost',
  phone: '415-123-4567',
)

puts strict_verifiable_address

File: official/docs/go/address/strict_verify_address_on_create.go
package example

import (
    "os"

    "github.com/EasyPost/easypost-go/v2"
)

func main() {
	apiKey := os.Getenv("EASYPOST_API_KEY")
    client := easypost.New(apiKey)

	address, err := client.CreateAddress(
        &easypost.Address{
            Street1: "417 MONTGOMERY ST",
            Street2: "FLOOR 5",
            City:    "SAN FRANCISCO",
            State:   "CA",
            Zip:     "94104",
            Country: "US",
            Company: "EasyPost",
            Phone:   "415-123-4567",
        },
        &easypost.CreateAddressOptions{
            VerifyStrict: []string{"true"},
        },
    )

	fmt.Println(address) 
}